src/vuetify-week-scheduler.vue

```diff
@@ -107,6 +140,18 @@ export default {
       newPeriod: null,
       draggingPeriod: null,
       resizingPeriod: null,
+      showEditMenu: false,
+      x: 0,
+      y: 0,
+      editEvent: null,
+      colors: [
+        "#F44336",
+        "#3F51B5",
+        "#9C27B0",
+        "#FF9800",
+        "#4CAF50",
+        "#FFEB3B",
+      ],
```

Comment:
I think now we could make this become a configuration option too like it was in original implementation